### PR TITLE
Fix and improve docs: oCIS deployment in K8s

### PR DESCRIPTION
**Pull request by saw-jan**
Tried to deploy oCIS using K8s as per the documentation testing: https://github.com/owncloud/ocis/issues/4595.
Found some invalid commands and step orders. Also, did some yaml formatting

### CODE REVIEW

File: modules/ROOT/examples/deployment/container/orchestration/nginx-ingress-tab-1.yaml [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
+        - ocis.kube.owncloud.test
+      secretName: ocis-tls-certificate
 
-  extraResources:
-    - |
-      apiVersion: cert-manager.io/v1
-      kind: Issuer
-      metadata:
-        name: ocis-certificate-issuer
-        namespace: ocis-namespace
```
Comment (saw-jan) [inline modules/ROOT/examples/deployment/container/orchestration/nginx-ingress-tab-1.yaml:19]: I have removed `namespace: ocis-namespace` entry. If that is to include then we might need to document somewhere a command to add namespace
such as `minikube kubectl -- create namespace ocis-namespace`

File: modules/ROOT/examples/deployment/container/orchestration/nginx-ingress-tab-1.yaml [hunk 2]
```diff
@@ -1,29 +1,28 @@
-externalDomain: ocis.owncloud.test
+externalDomain: ocis.kube.owncloud.test
```
Comment (saw-jan) [inline modules/ROOT/examples/deployment/container/orchestration/nginx-ingress-tab-1.yaml:1]: At the end we are adding `ocis.kube.owncloud.test` to the `/etc/hosts`. So, I thought to make it consistent everywhere.

File: modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc [hunk 3]
```diff
@@ -214,6 +212,22 @@ include::{ocis-charts-raw-url}{tab_3}{kube-versions-url}[]
...
+minikube start --kubernetes-version=v1.24.6
+----
+
+. Enable the minikube ingress plugin, which acts like a reverse proxy for your cluster:
++
+[source,bash]
+----
+minikube addons enable ingress
+----
+
```
Comment (saw-jan) [inline modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc:230]: Need to start minikube cluster before deploying the chart

File: modules/ROOT/examples/deployment/container/orchestration/nginx-ingress-tab-1.yaml (hunk 1 above)
Comment (saw-jan) [reply modules/ROOT/examples/deployment/container/orchestration/nginx-ingress-tab-1.yaml:19]: Oh! I remember that I got a namespace not found message.
Please, confirm from your side

**Comment by mmattel** [comment]
@saw-jan I will continue if @wkloucek agrees to all of your proposals

@wkloucek pls respond to the open topics/resolve them so I know the state

File: modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc [hunk 4]
```diff
@@ -626,11 +624,18 @@ endif::[]
 
 ==== Apply Chart Changes
 
-* Install all changes defined in your _own_ `values.yaml` file:
+* Apply all changes defined in your _own_ `values.yaml` file:
++
+[source,bash]
+----
+helm upgrade ocis ./charts/ocis --values values.yaml
```
[the remark this answers is not in the document]
Comment (saw-jan) [reply modules/ROOT/pages/deployment/container/orchestration/orchestration.adoc:631]: Yeah, that's good.
I was just continuing from the steps above and `helm install` gave me an error so I changed to upgrade.

**Review by mmattel — APPROVED** [review]

**Pull request merged**